### ISSUE TITLE
Various jdk14 slave jenkins/slave failures

### ISSUE THREAD

**Issue by andrew-m-leonard** [AdoptOpenJDK/openjdk-infrastructure#1380]
https://ci.adoptopenjdk.net/view/Failing%20Builds/job/build-scripts/job/jobs/job/jdk14u/job/jdk14u-linux-ppc64le-hotspot/93/console
```
21:37:19  Caused by: hudson.plugins.git.GitException: Command "git clean -fdx" returned status code 143:
21:37:19  stdout: 
21:37:19  stderr: 
```
https://ci.adoptopenjdk.net/view/Failing%20Builds/job/build-scripts/job/jobs/job/jdk14u/job/jdk14u-windows-x64-hotspot/96/console
```
21:52:30  wrapper script does not seem to be touching the log file in J:/jenkins/tmp@tmp/durable-e3e5e0fc
21:52:30  (JENKINS-48300: if on an extremely laggy filesystem, consider -Dorg.jenkinsci.plugins.durabletask.BourneShellScript.HEARTBEAT_CHECK_INTERVAL=86400)
```

https://ci.adoptopenjdk.net/view/Failing%20Builds/job/build-scripts/job/jobs/job/jdk14u/job/jdk14u-windows-x64-openj9-windowsXL/102/console
https://ci.adoptopenjdk.net/view/Failing%20Builds/job/build-scripts/job/jobs/job/jdk14u/job/jdk14u-windows-x86-32-hotspot/99/console
```
21:27:23  	Suppressed: jenkins.util.io.CompositeIOException: Unable to remove directory C:\workspace\openjdk-build\workspace\build\src\build\windows-x86_64-normal-server-release\jdk\gensrc\java\nio with directory contents: [C:\workspace\openjdk-build\workspace\build\src\build\windows-x86_64-normal-server-release\jdk\gensrc\java\nio\_the..]
```

**Comment by andrew-m-leonard** [comment]
I think something "transient" across the builds must have happened last night, re-build seems to be working fine

**Comment by Willsparker** [comment]
The top one is probably due to a connection issue: https://issues.jenkins-ci.org/browse/JENKINS-36269 . It looks like it hit the timeout of 10 minutes.

The middle one; Supposedly according to [this](https://stackoverflow.com/questions/56292105/jenkins-suddenly-showing-wrapper-script-does-not-seem-to-be-touching-the-log-fi), it was possibly an issue with the Jenkins master node? 

The bottom two have just been fixed : #1379

**Comment by Willsparker** [comment]
> I think something "transient" across the builds must have happened last night, re-build seems to be working fine

That would line up; A few of my OSs in a `VPC` run all had Jenkins connection issues at 23:14:02 last night. Seems alright this morning
